uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -234,7 +234,7 @@ sub getSessionID {
 	# fixme ask Jon ... is this still useful?
     # determine the centerID and new visit number (which is now deprecated) if getPSC() failed.
 	if($centerID == 0) {
-            $query = "SELECT IFNULL(MAX(VisitNo), 0)+1 AS newVisitNo, CenterID FROM session WHERE CandID=".$dbh->quote($subjectIDref->{'CandID'})." GROUP BY CandID";
+            $query = "SELECT IFNULL(MAX(VisitNo), 0)+1 AS newVisitNo, CenterID FROM session WHERE CandID=".$dbh->quote($subjectIDref->{'CandID'})." GROUP BY CandID newVisitNo, CenterID";
```

Comment:
@Jkat 
I think you have a typo in this query!
but out of curiosity, when would you exercise this part of the loop? is it when you create the session from the back end? because this is not the default setting (projects like CCNA and IBIS do not use this).